api_app/script_analyzers/observable_analyzers/inquest.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import requests
+import time
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers.classes import ObservableAnalyzer
+from intel_owl import secrets
+
+
+logger = logging.getLogger(__name__)
+
+
+class InQuest(ObservableAnalyzer):
+    base_url: str = "https://labs.inquest.net"
+    max_tries: int = 10
+    poll_distance: int = 10
+
+    def set_config(self, additional_config_params):
+        self.api_key_name = additional_config_params.get(
+            "api_key_name", "INQUEST_API_KEY"
+        )
+        self.__api_key = secrets.get_secret(self.api_key_name)
+        self.analysis_type = additional_config_params.get("inquest_analysis", "dfi_search")
+        self.observable_classification = additional_config_params.get("inquest_analysis", "details")
```

Comment:
`self.observable_classification` comes from the base analyzer i.e. `ObservableAnalyzer` we don't need to manually set it.